scribejava-apis/src/main/java/com/github/scribejava/apis/service/MicrosoftAzureActiveDirectoryService.java

```diff
@@ -8,18 +8,18 @@
 
 public class MicrosoftAzureActiveDirectoryService extends OAuth20Service {
 
-    private static final String ACCEPTED_FORMAT
-            = "application/json; odata=minimalmetadata; streaming=true; charset=utf-8";
+    private final String acceptedFormat;
 
     public MicrosoftAzureActiveDirectoryService(DefaultApi20 api, String apiKey, String apiSecret, String callback,
             String scope, String state, String responseType, String userAgent, HttpClientConfig httpClientConfig,
-            HttpClient httpClient) {
+            HttpClient httpClient, String acceptedFormat) {
         super(api, apiKey, apiSecret, callback, scope, state, responseType, userAgent, httpClientConfig, httpClient);
+        this.acceptedFormat = acceptedFormat;
     }
 
     @Override
     public void signRequest(String accessToken, OAuthRequest request) {
```

Comment:
This method isn't good example of clean code. Why we set a header in a method called "signRequest". I didn't change that though...

Reply:
Hm. You're right, isn't very clean